apps/design/frontend/src/ballot_screen.tsx

```diff
@@ -307,10 +309,12 @@ export function BallotScreen(): JSX.Element | null {
                     height: '100%',
                   }}
                 >
-                  <Card color="danger">
-                    Error:{' '}
-                    {ballotResult.err().message ?? 'Something went wrong'}
-                  </Card>
+                  <Callout color="danger" icon="Danger">
+                    <span>
+                      Contest &quot;{err.contest.title}&quot; was too long to
+                      fit on the page. Try a longer paper size.
```

Comment:
nit: if the user is already trying the longest paper size what are their options?

Reply:
Contact support. I figured they'd resort to that by default so didn't include in the message